readest: 0.9.23 -> 0.9.28

ddogfoodd opened the pull request:
Changelog: https://github.com/readest/readest/releases/tag/v0.9.28
Diff: https://github.com/readest/readest/compare/v0.9.23...v0.9.28

also switching to `final.Attrs`

add @eljamm as maintainer

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

ddogfoodd committed:
readest: 0.9.23 -> 0.9.25

Changelog: https://github.com/readest/readest/releases/tag/v0.9.25
Diff: https://github.com/readest/readest/compare/v0.9.23...v0.9.25

0xda157 reviewed (approved):
diff lgtm

ddogfoodd committed:
readest: 0.9.25 -> 0.9.26

Changelog: https://github.com/readest/readest/releases/tag/v0.9.26
Diff: https://github.com/readest/readest/compare/v0.9.25...v0.9.26

awwpotato commented on pkgs/by-name/re/readest/package.nix:
```diff
@@ -19,13 +19,13 @@
 
 rustPlatform.buildRustPackage rec {
```
Switching to finalAttrs would be nice but definitely not a blocker

0xda157 reviewed (approved):
diff still lgtm

ddogfoodd committed:
readest: use finalAttrs

eljamm commented:
I'm interested in picking up this package, so could you please add me to its maintainers, @ddogfoodd?

PS: version [0.9.27](https://github.com/readest/readest/releases/tag/v0.9.27) just came out.

ddogfoodd committed:
readest: 0.9.26 -> 0.9.27

Changelog: https://github.com/readest/readest/releases/tag/v0.9.27
Diff: https://github.com/readest/readest/compare/v0.9.26...v0.9.27

ddogfoodd committed:
readest: add eljamm as maintainer

as requested in
https://github.com/NixOS/nixpkgs/pull/392500#issuecomment-2763240138

kirillrdy commented:
you can squash if you wish, it's not required. main requirement it that each commit must eval

eljamm commented:
> Do I have to squash commits in this case? Or just for trivial changes like removing whitespace or formatting?

I think it's fine here since each commit describes a unique/logical change. I've seen separate commits for formatting, before, but for small fixups and changes that belong together, it's best to squash.

eljamm reviewed (approved):
LGTM, thanks!

eljamm commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 392500`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>

eljamm commented:
It works for me too, locally. Guess this was just the fault of the [GitHub actions nixpkgs-review](https://github.com/eljamm/nixpkgs-reviewing-workspace) I was using. Tried it with [another one](https://github.com/JohnRTitor/nixpkgs-review-gha) and it's currently building it.

eljamm reviewed (approved):
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 392500`

Logs: https://github.com/eljamm/nixpkgs-review-gha/actions/runs/14144963714


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>

0xda157 reviewed (approved):
lgtm

ddogfoodd committed:
readest: 0.9.27 -> 0.9.28

Changelog: https://github.com/readest/readest/releases/tag/v0.9.28
Diff: https://github.com/readest/readest/compare/v0.9.27...v0.9.28

eljamm reviewed (approved):
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 392500`

Logs: https://github.com/eljamm/nixpkgs-review-gha/actions/runs/14150002762


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>
